app/views/foods/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<section class='foods-wrapper flex-items'>
+  <div class="add-food flex-items">
+    <div class="food-items-header">
+      <h1>Foods</h1>
+      <%= link_to "Add food", new_food_path, class:"button"%>
+    </div>
```

Comment:
- [Option] You have don an interesting project, Team. For the price though, I would suggest if it would be possible to add a float number as a value. this makes your app the real world. 

![image](https://user-images.githubusercontent.com/69077061/233673503-eea2adfc-4f27-4910-b163-81d425f97333.png)